package.json

```diff
@@ -45,7 +45,7 @@
     "shelljs": "^0.8.5",
     "sync-request": "4.1.0",
     "typed-rest-client": "^1.8.9",
-    "typescript": "4.0.2",
+    "typescript": "^4.0.2",
```

Comment:
change needed?

Reply:
It's auto generated after running `Node make.js build --task HelmDeployV0` command.

Reply:
I checked no errors if I removed this change, so it may be safe to remove it.